neo/SmartContract/Native/Votes/VoteContract.cs

```diff
@@ -0,0 +1,505 @@
+#pragma warning disable IDE0051
+#pragma warning disable IDE0060
+
+using Neo.Cryptography;
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.SmartContract.Native.Tokens;
+using Neo.SmartContract.Native.Votes.Model;
+using Neo.VM;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.Serialization.Formatters.Binary;
+using Neo.SmartContract.Native.Votes.Interface;
+using VMArray = Neo.VM.Types.Array;
+
+namespace Neo.SmartContract.Native
+{
+    public sealed class VoteContract : NativeContract
+    {
+        public override string ServiceName => "Neo.Native.VoteContract";
+        private const byte Prefix_CreateVote = 100;
```

Comment:
format, add more one line above this line, two different region should split by one line.